### ISSUE TITLE
Error when trying the example

### ISSUE THREAD

**Issue by autonomobil** [JuliaPOMDP/ARDESPOT.jl#17]
I tried to use the example: 
```
using POMDPs, POMDPModels, POMDPSimulators, ARDESPOT

pomdp = TigerPOMDP()

solver = DESPOTSolver(bounds=(-20.0, 0.0))
planner = solve(solver, pomdp)

for (s, a, o) in stepthrough(pomdp, planner, "sao", max_steps=10)
    println("State was $s,")
    println("action $a was taken,")
    println("and observation $o was received.\n")
end
```

But I get this error message:

```
┌ Warning: uncrecognized symbol sao in step iteration specification sao.
└ @ POMDPSimulators ~/.julia/packages/POMDPSimulators/nMXAP/src/stepthrough.jl:145
ERROR: LoadError: type NamedTuple has no field sao
Stacktrace:
 [1] getindex at ./namedtuple.jl:107 [inlined]
 [2] macro expansion at /home/mo/.julia/packages/POMDPSimulators/nMXAP/src/stepthrough.jl:109 [inlined]
 [3] out_tuple at /home/mo/.julia/packages/POMDPSimulators/nMXAP/src/stepthrough.jl:103 [inlined]
 [4] iterate(::POMDPSimulators.POMDPSimIterator{:sao,TigerPOMDP,DESPOTPlanner{TigerPOMDP,Tuple{Float64,Float64},MemorizingSource{Random.MersenneTwister},Random.MersenneTwister},ParticleFilters.BasicParticleFilter{TigerPOMDP,TigerPOMDP,ParticleFilters.LowVarianceResampler,Random.MersenneTwister,Array{Bool,1}},Random._GLOBAL_RNG,ParticleFilters.ParticleCollection{Bool},Bool}, ::Tuple{Int64,Bool,ParticleFilters.ParticleCollection{Bool}}) at /home/mo/.julia/packages/POMDPSimulators/nMXAP/src/stepthrough.jl:98
 [5] iterate(::POMDPSimulators.POMDPSimIterator{:sao,TigerPOMDP,DESPOTPlanner{TigerPOMDP,Tuple{Float64,Float64},MemorizingSource{Random.MersenneTwister},Random.MersenneTwister},ParticleFilters.BasicParticleFilter{TigerPOMDP,TigerPOMDP,ParticleFilters.LowVarianceResampler,Random.MersenneTwister,Array{Bool,1}},Random._GLOBAL_RNG,ParticleFilters.ParticleCollection{Bool},Bool}) at /home/mo/.julia/packages/POMDPSimulators/nMXAP/src/stepthrough.jl:86

```

**Comment by zsunberg** [comment]
Hi @autonomobil , thanks for reporting. The code you copied is a little out of date and `stepthrough` no longer supports "sao" squashed together. Instead, you have to use "s,a,o". Would you mind letting me know where you found that example so I can go back and fix it?

**Comment by autonomobil** [comment]
Ah, okay that make sense. It is in the main README.md. The following code works:

```
using POMDPs, POMDPModels, POMDPSimulators, ARDESPOT

pomdp = TigerPOMDP()

solver = DESPOTSolver(bounds=(-20.0, 0.0))
planner = solve(solver, pomdp)

for (s, a, o) in stepthrough(pomdp, planner, "s,a,o", max_steps=10)
    println("State was $s,")
    println("action $a was taken,")
    println("and observation $o was received.\n")
end
```

I created a pull request for the small type fix in the README.md
https://github.com/JuliaPOMDP/ARDESPOT.jl/pull/18

**Comment by zsunberg** [comment]
This was fixed with #18
